crates/sui-types/src/effects/effects_v2.rs

```diff
@@ -595,4 +607,6 @@ pub enum UnchangedSharedKind {
     ReadDeleted(SequenceNumber),
     /// Shared objects in cancelled transaction. The sequence number embed cancellation reason.
     Cancelled(SequenceNumber),
+    /// Read of a per-epoch config object that should remain the same during an epoch.
+    PerEpochConfig,
```

Comment:
this requires a new protocol version, right?

Reply:
The per epoch config object is only checked when coin deny list v2 is enabled, and hence is already behind a new protocol version.

Reply:
got it